packages/backend-common/config.d.ts

```diff
@@ -72,10 +75,23 @@ export interface Config {
       /** Default database client to use */
       client: 'better-sqlite3' | 'sqlite3' | 'pg';
       /**
-       * Base database connection string or Knex object
+       * Base database connection string or Knex (PG) object
```

Comment:
```suggestion
       * Base database connection string, or object with individual connection properties
```

or somesuch - it's not defined by knex actually, and either way it's not PG specific